tests/foreman/destructive/test_installer.py

```diff
@@ -179,3 +179,44 @@ def test_positive_installer_puma_worker_count(target_sat):
             assert f'cluster worker {i}' in result.stdout
         else:
             assert f'cluster worker {i}' not in result.stdout
+
+
+def test_handle_invalid_certificate(cert_setup_destructive_teardown):
```

Comment:
to be consistent with naming around robottelo I suggest `test_negative_handle...`

Reply:
Ok, let it be `negative`...